packages/icons/src/neutral/home.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Keep only the `xmlns` and the `viewBox` attributes, since the other ones are irrelevant.

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
```